### DISCUSSION TITLE
WebDAV protocol

### DISCUSSION THREAD

**Discussion by tomtom68de** [rejetto/hfs#761] [Q&A]
Hello everyone,
I read some time ago (June 2024) that WebDAV was being considered. Is there any news on this?
Thanks and best regards
Tom

**Comment by rejetto** [comment]
hey!  no news.
i won't work on it until the 0.XX version series is over, that should happen before the end of the year, hopefully. 
you can subscribe #261 as i will surely keep you posted there. It's intentionally left open.

**Reply by tomtom68de** [reply]
Hi rejetto,
thanks for the information. I've subscribed to #261 and fingers crossed that it can be started soon. Is there anything I can do to help speed up the current process?
Regards

**Comment by rejetto** [comment]
what you may do is to make a plugin that adds webdav integration.
it takes a node.js programmer, and it's not trivial even for one, because it's not like making a new server; it must work "within" the existing one.

**Comment by mitobin** [comment]
I mentioned this issue at the end of last year and you said it could be deployed by the end of this year. Hope you complete it soon. best regards!

**Comment by rejetto** [comment]
you say "deployed", but i said ["work on it"](https://github.com/rejetto/hfs/discussions/417#discussioncomment-8205703). 😉
There's no ETA for the moment.
I can only hope that the "final" hfs 3 will be ready before the end of the year, and it's not planned to include webdav.
I'm doing this in my spare time.
